index.html

```diff
@@ -132,22 +131,22 @@ <h3>Multi-Post Storiese</h3>
               ever since the 1500s, when an unknown printer took a standard
               dummy text.
             </p>
-            <div id="tb3">
-              <span id="a">HTML</span>
-              <span id="b">CSS</span>
-              <span id="c">BOOTSTRAP</span>
-              <span id="d">RUBY</span>
+            <div id="tb2">
+              <span id="a2">HTML</span>
+              <span id="b2">CSS</span>
+              <span id="c2">BOOTSTRAP</span>
+              <span id="d2">RUBY</span>
             </div>
-            <button>See Project</button>
+            <button type="reset">See Project</button>
```

Comment:
- [ ] `required`
- [ ] Kindly add a hover on `all the buttons` `why?` so that someone is able to tell that its a button when they hover on